src/Ombi.Core/Rule/Rules/Search/ExistingRule.cs

```diff
@@ -60,6 +62,8 @@ public async Task<RuleResult> Execute(SearchViewModel obj)
 
                     request.Requested = true;
                     request.Approved = tvRequests.ChildRequests.Any(x => x.Approved);
+                    request.Denied = tvRequests.ChildRequests.Any(x => x.Denied ?? false);
```

Comment:
I'm not sure how I feel about this. Since we can have many child requests for one show. If one of them is denied it will seem like the whole show has been denied.

Reply:
I agree, I took the line above as an example. I believed that if `Approved` was initialized like that, it would be OK for `Denied` as well. 

One solution would be to have the same kind of mechanism as `request.FullyAvailable` and `request.PartlyAvailable`, for `Approved` and `Denied`. Not sure about the complexity, since it would probably affect the UI. What do you think?

Reply:
Good point, we don't show the approved as part of the UI. But good point.

I think Denied in this case is probably fine, but can we remove the Denied Reason? since that could vary if there are multiple denies, if someone wants to see the reason for denial then they can view the child request.

Reply:
Either that, or we format a comma-separated list of all reasons: this might get messy in some cases, but at least a relevant value is returned? 

Unless you say otherwise, I'll remove this line. Waiting for your final word to push. 🙂

Reply:
Yeah I think that would get quite messy if we had 10 denied requests with 50 chars each for example! :)